server/database/models/User.js

```diff
@@ -1,13 +1,13 @@
 const mongoose = require('mongoose');
 
 const userSchema = new mongoose.Schema({
-  _id: String,
-  userName: {
+  email: {
     type: String,
     required: true
   },
-  email: String,
-  picture: String
+  name: String,
+  salt: String,
```

Comment:
Do we still need the salt to be stored

Reply:
Not sure yet, will look into it further. Currently keeping it in case we do.

Reply:
deciding to keep and do more research in the future

Reply:
I looked into it. The salt is stored internally in the hash as the 22 characters after the third dollar sign, starting from the left (supposedly). I'm 99% sure we don't need it